src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -308,7 +324,6 @@ export const useSwapper = () => {
           sellAmount: amount,
           sellAsset,
           buyAsset,
```

Comment:
bip44params?

Reply:
and accountType

Reply:
Nice catch, both are already here with the old logic but I've moved them to bip44Params and accountType from metadata - 868eebc98f470a269703342b8bb093efd100c2be